reactive/calico.py

```diff
@@ -672,6 +677,22 @@ def repull_calico_node_image():
     remove_state('calico.service.installed')
 
 
+@when('calico.service.installed', 'calico.pool.configured')
+def disable_vxlan_tx_checksumming():
+    '''Workaround for https://github.com/projectcalico/calico/issues/3145'''
+    config = charm_config()
+
+    if config['disable-vxlan-tx-checksumming'] and config['vxlan'] != 'Never':
+        cmd = ['ethtool', '-K', 'vxlan.calico', 'tx-checksum-ip-generic',
+               'off']
```

Comment:
Is there any cost to doing this every hook, or should we try to be smarter and limit it to only install and machine restart?

Reply:
There is a time cost, of course, but not a huge one. Ideally we would do something smarter here, but I'm hopeful that this will only be a temporary workaround - one that we can remove after we upgrade to Calico 3.16.x.